docs/source/concepts/ledger-model/ledger-privacy.rst

```diff
@@ -28,18 +28,24 @@ Generalizing this, **observers** are parties who might not be bound by the contr
 For example, Alice should be an observer of the `PaintOffer`, such that she is made aware that the offer exists.
 
 Signatories are already determined by the contract model discussed so far.
-The full **contract model** additionally specifies the observers on each contract.
-A **stakeholder** of a contract (according to a given
-contract model) is then either a signatory or an observer on the contract.
-Note that in Daml, as detailed :ref:`later <da-model-daml>`, controllers specified using simple syntax are automatically made observers whenever possible.
+The full **contract model** additionally specifies the **contract observers** on each contract.
+A **stakeholder** of a contract (according to a given contract model) is then either a signatory or a contract observer on the contract.
+Note that in Daml, as detailed :ref:`later <da-model-daml>`, controllers specified using simple syntax are automatically made contract observers whenever possible.
 
-In the graphical representation of the paint offer acceptance below, observers who are not signatories are indicated by an underline.
+In the graphical representation of the paint offer acceptance below, contract observers who are not signatories are indicated by an underline.
 
 .. https://www.lucidchart.com/documents/edit/ea40a651-a2e0-4365-ae7d-4cee8cd07071/0
 .. image:: ./images/stakeholders-paint-offer.svg
    :align: center
    :width: 60%
 
+Choice Observers
+++++++++++++++++
+
+In addition to contract observers, we also have **choice observers** on individual contract choices.
+Choice observers get to see a specific exercise on a contract, and to view its consequences.
```

Comment:
```suggestion
Choice observers get to see a all exercises of a specific choice on a contract, and to view its consequences.
```

Reply:
Right now the ledger model doesn't include a notion of a choice. I do want to add it when I find a bit of spare time, but until then I would maybe not mention them here.

Reply:
A short blurb choice observers should also be added to the "DAML: Defining Contract Models Compactly" section

Reply:
@cocreature  I think a choice observer can depend on the arguments to the choice, right? So they really are observers to a specific exercise, not to every exercise on the choice.

Reply:
good point, I guess it might still be useful to state somewhere that they are defined by choice but that brings up to @oggy-’s point that choices aren’t included in here at all so keeping it at exercises should be perfectly fine.

Reply:
Maybe we should have called them "exercise observer"... In any case I think @sofiafaro-da 's phrasing was good.